spark/v3.5/spark/src/main/java/org/apache/iceberg/spark/source/SparkScan.java

```diff
@@ -220,7 +245,15 @@ public CustomMetric[] supportedCustomMetrics() {
       new ScannedDataManifests(),
       new SkippedDataManifests(),
       new ScannedDataFiles(),
-      new SkippedDataFiles()
+      new SkippedDataFiles(),
```

Comment:
Once we add blocks above, can we match the order of these variables so that it is easier to cross check?